spec/providers/atmosphere/local_pdp_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'rails_helper'
+
+describe Atmosphere::LocalPdp do
+  context '#can_manage?' do
+    let(:user) { build(:user, id: 1) }
+    subject { Atmosphere::LocalPdp.new(user) }
+
+    it 'allows to manage owned appliance types' do
+      at = build(:appliance_type, user_id: user.id)
```

Comment:
build is enough here? In `can_manage?` function you are doing db query, thus I'm not sure how it works since this AT does not exist in db.